backend/database/migrations/2021_08_02_005748_create_followings_table.php

```diff
@@ -16,8 +16,9 @@ public function up()
         Schema::create('followings', function (Blueprint $table) {
             $table->id();
             $table->foreignId('student_id');
-            $table->bigInteger('followers');
-            $table->string('description');
+            $table->bigInteger('student_follower');
+            $table->bigInteger('student_follow');
+            $table->string('type');
```

Comment:
Do not edit the pushed migrations, rather create a new migration when updating a column
refer to this video:
https://www.youtube.com/watch?v=d1q2TGTGfzw&ab_channel=Ruby

Reply:
I see, I understand Sir. I'll work on it.

Reply:
i dont mind this for now but in actual development environment you will need to create separate migration for updating columns, adding columns, and removing columns. The reason for this is to prevent running `migrate:fresh` which clears your data just to remigrate all migration file. And if we run it in production it will remove all existing data from our application which we don't want since users will already be using the application.